### PR TITLE
Add random UA

### PR DESCRIPTION
**Pull request by leon332157**
Two ways to generate random UA, use 'fake-useragent' module, which is reound_FAKE_UA.py or use a list of UA, which is user_agent and random choice in rebound.py

### CODE REVIEW

File: rebound/rebound_FAKE_UA.py [hunk 1]
```diff
@@ -0,0 +1,775 @@
...
+import os
+from bs4 import BeautifulSoup
+import requests
+from queue import Queue
+from subprocess import PIPE, Popen
+from threading import Thread
+import webbrowser
+import time
+from urwid.widget import (BOX, FLOW, FIXED)
+from fake_useragent import UserAgent
```
Comment (leon332157) [inline rebound/rebound_FAKE_UA.py:26]: Needs change in requirements.txt

**Comment by shobrook** [comment]
Hey, I'd be willing to merge this if you resolve the conflicts and fix some of the style changes you made (it looks like your editor might have AutoPEP8 installed?).

**Comment by leon332157** [comment]
@shobrook yes, I use PyCharm

**Comment by leon332157** [comment]
@shobrook So do you prefer using UA package or use a list?

**Comment by shobrook** [comment]
I think using a list is the better option. It's more lightweight.

**Comment by leon332157** [comment]
@shobrook ok got it

**Comment by leon332157** [comment]
@shobrook I'll start a new PR because I deleted my fork

**Pull request closed without merging**